projects/plugins/jetpack/_inc/client/components/jetpack-loading-icon/index.jsx

```diff
@@ -13,10 +13,10 @@ import { imagePath } from 'constants/urls';
  */
 import './style.scss';
 
-export const LoadingCard = () => {
+export const JetpackLoadingIcon = () => {
 	return (
-		<div className="jp-recommendations-loading-card">
-			<img src={ imagePath + '/jetpack-logomark-blue.svg' } alt="Loading recommendations" />
+		<div className="jp-loading-icon">
+			<img src={ imagePath + '/jetpack-logomark-blue.svg' } alt="Loading..." />
```

Comment:
I think it would be cool to get the `alt` text as a prop, too. So we can use custom alt texts for different contexts

Reply:
Yes! Great catch! 👏